tsconfig.json

```diff
@@ -13,9 +13,8 @@
       "dom",
       "es6"
     ],
-    "types": [
-      "jasmine",
-      "node"
+    "typeRoots": [
```

Comment:
Ok, looking at this a little more.  Reading over the "@types, typeRoots and types" section of https://www.typescriptlang.org/docs/handbook/tsconfig-json.html, it seems you've put back in what is the default behavior.

Since you're importing `fontFaceObjserver` using the module syntax, I would've expected this sentence to be true.  

"Keep in mind that automatic inclusion is only important if you’re using files with global declarations (as opposed to files declared as modules). If you use an import "foo" statement, for instance, TypeScript may still look through node_modules & node_modules/@types folders to find the foo package."